qiskit/qasm3/ast.py

```diff
@@ -596,11 +596,30 @@ class DefaultCase(Expression):
     """An object representing the `default` special label in switch statements."""
 
 
-class SwitchStatement(Statement):
-    """AST node for the proposed 'switch-case' extension to OpenQASM 3."""
+class SwitchStatementV1(Statement):
```

Comment:
I'd be inclined to call this something a bit more descriptive like `SwitchStatementPreview` or `ExperimentalSwitchStatement`. When I see V1 here I feel like this means there is another stable revision in the spec.

Reply:
Yeah, I took the V1 from the name in the `ExperimentalFlags` enum, but `SwitchStatementPreview` or something is better.  Deon in 089165901d.